### PR TITLE
fix showconnections in new profiles

### PR DESCRIPTION
**Pull request by Masterjoona**
fixes showconnections for simplified profiles

fixing reviewdb and permviewer is as easy as this but im kinda stupid and couldnt figure out how to make scrolling work 😭

### CODE REVIEW

**Comment by Vendicated** [comment]
thanks! looks good codewise

imo the margin above the Connections header is too much though, maybe we should reduce it
image

**Comment by Masterjoona** [comment]
yeah it caught my eye also

**Comment by Masterjoona** [comment]
image
margin gone

File: src/plugins/showConnections/index.tsx [hunk 1]
```diff
@@ -188,6 +190,13 @@ export default definePlugin({
                 match: /\(0,\i\.jsx\)\(\i\.\i,\{\}\).{0,100}setNote:(?=.+?channelId:(\i).id)/,
                 replace: "$self.profilePanelComponent({ id: $1.recipients[0] }),$&"
             }
+        },
+        {
+            find: "autoFocusNote:!0})",
+            replacement: {
+                match: /{autoFocusNote:!1}\)}\)(?<=user:(\i),bio:null==(\i)\?.+?)/,
+                replace: "$&,$self.profilePopoutComponent({ user: $1, displayProfile: $2, useDiv: !0 })"
```
Comment (Vendicated) [inline src/plugins/showConnections/index.tsx:198]:
```suggestion
                replace: "$&,$self.profilePopoutComponent({ user: $1, displayProfile: $2, compactSpacing: true })"
```

**Review by Vendicated — APPROVED** [review]
looks good now, thanks! :3

**Pull request merged**